### DISCUSSION TITLE
Add Controller Support

### DISCUSSION THREAD

**Discussion by KoriusX** [Leon280698/CT#18] [Ideas]
Steam has profiles for mapping controller inputs to keyboard and mouse, but these lack aim assist and other small touches that go a long way towards a smoother experience on controller. If at all possible, I'm sure myself and many others would appreciate such an addition!

**Comment by leon-bckl** [comment]
Personally, not something I’m interested in doing but I’m open to contributions.

**Comment by Flavio93Zena** [comment]
+1 on this one, I was only able to find this page [https://community.pcgamingwiki.com/files/file/659-gamepad-files-republic-commando/](url) but was not able to get it to work at all, all commands are pressed randomly at once. The game works beautifully if I start it without plugging the controller in, but if I do plug it in, it's a mess.
Still, thanks for the mod.
